cfme/tests/test_db_migrate.py

```diff
@@ -134,9 +244,17 @@ def test_db_migrate(app_creds, temp_appliance_extended_db, db_url, db_version, d
         '"u = User.find_by_userid(\'admin\'); u.password = \'{}\'; u.save!"'
         .format(app.user.credential.secret))
     assert rc == 0, "Failed to change UI password of {} to {}:" \
-                    .format(app.user.credential.principal, app.user.credential.secret, out)
+        .format(app.user.credential.principal, app.user.credential.secret, out)
     app.server.login(app.user)
 
+    app.set_pglogical_replication(replication_type=':remote')
+    app2.set_pglogical_replication(replication_type=':global')
+    app2.add_pglogical_replication_subscription(app.hostname)
+
+    def is_provider_replicated(app, app2):
+        assert set(app.managed_provider_names) == set(app2.managed_provider_names)
```

Comment:
**REQUIRED**
I believe it's a typo, there should be `return` instead of `assert`.